src/applications/vaos/components/layout/DetailPageLayout.jsx

```diff
@@ -209,6 +198,12 @@ export default function DetailPageLayout({
 }) {
   const { id } = useParams();
   const { facility } = useSelector(state => selectFacility(state, id));
+  const travelPayViewClaimDetails = useSelector(state =>
+    selectFeatureTravelPayViewClaimDetails(state),
+  );
+  const travelPaySubmitMileageExpense = useSelector(state =>
+    selectFeaturetravelPaySubmitMileageExpense(state),
```

Comment:
Minor nit:
```suggestion
    selectFeatureTravelPaySubmitMileageExpense(state),
```

Reply:
Minor, but we typically prefix feature flag variable with the prefix, "feature" i.e. featureTravelPaySubmitMileageExpense.